modules/third_party_perception/tools/conversion_smartereye.cc

```diff
@@ -0,0 +1,273 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file
+ */
+
+#include <map>
+#include <vector>
+#include <iostream>
+#include <cmath>
+
+#include "cyber/common/log.h"
+#include "modules/common/configs/config_gflags.h"
+
+#include "modules/third_party_perception/common/third_party_perception_gflags.h"
+#include "modules/third_party_perception/common/third_party_perception_util.h"
+#include "modules/third_party_perception/tools/conversion_smartereye.h"
+
+/**
+ * @namespace apollo::third_party_perception::conversion_smartereye
+ * @brief apollo::third_party_perception
+ */
+namespace apollo {
+namespace third_party_perception {
+namespace conversion_smartereye {
+
+using apollo::canbus::Chassis;
+using apollo::localization::LocalizationEstimate;
+using apollo::perception::PerceptionObstacle;
+using apollo::perception::PerceptionObstacles;
+using Point = apollo::common::Point3D;
+
+std::map<std::int32_t, apollo::hdmap::LaneBoundaryType_Type>
```

Comment:
If the mapping are the same, we can probably share this one and the one in mobileye.